env-configuration/README.md

```diff
@@ -7,6 +7,9 @@ We use the same tool here to produce a Docker image that is easily compatible wi
 
 The resulting Docker images are available from Docker Hub: [`turinginst/quipp-env`](https://hub.docker.com/r/turinginst/quipp-env).
 
+## Running the QUIPP pipeline using a container
+
+**TODO**: *Similar to Running the pipeline header on main README*
```

Comment:
It seems like there should be a section here on how to use the docker image:
1) Which tag should be used (since no "latest" tag exists)
2) Should you run the container interactively e.g. `docker run -it turinginst/quipp-env:0.0.4`